t/032_cancel_consume_ack_exception.t

```diff
@@ -0,0 +1,40 @@
+use Test::More tests => 17;
+use strict;
+use warnings;
+
+use FindBin qw/$Bin/;
+use lib "$Bin/lib";
+use NAR::Helper;
+
+my $helper = NAR::Helper->new;
+
+ok $helper->connect, "connected";
```

Comment:
That's a good point. Originally it did, but then I discovered it didn't matter. But now the test is named wrong so I'll add it back.